integration/int_3Ri_2BhaRi2_3Re_2BhaRe3/deploy_ssh.yml

```diff
@@ -68,6 +84,22 @@
 #
 - hosts: docker-host
   tasks:
+    # PYTHON3SCL - When using python3 from RHSCL the selinux module is failing
+    #              as it expects an /usr/bin/python3 executable to exist
+    #              A PR has been submitted to the selinux pip module to fix
+    #              this issue.
+    - name: Creating link for python ansible_python_interpreter in /usr/bin
+      command: "ln -s {{ ansible_python_interpreter }} /usr/bin/"
+      ignore_errors: yes
+    # END PYTHON3SCL
+
+    - name: Installing pip modules
+      pip:
+        executable: /opt/rh/rh-python36/root/usr/bin/pip3
+        name:
+          - docker
+          - selinux
+
```

Comment:
also duplicated with the previous diff, this could be a task or a role, I do not know if it is possible to import tasks from within the same file...

Reply:
I have just added a new commit and removed the duplicated tasks.

Reply:
:ok:  :ok: